src/status_im/subs/wallet/wallet_connect.cljs

```diff
@@ -42,3 +42,8 @@
  :<- [:wallet-connect/session-proposer]
  (fn [proposer]
    (-> proposer :metadata :name)))
+
+(rf/reg-sub
+ :wallet-connect/current-proposal-account
+ (fn [db]
+   (-> db :wallet-connect/current-proposal-account)))
```

Comment:
Consider using the shorthand `:->` makes the code less verbose

Reply:
Thanks, I didn't know about this. But I changed the structure so this is not relevant now. I'll keep in mind, next time.